add-ons/group-view.js

```diff
@@ -210,9 +210,9 @@ function getCell(defaultGetCell, config, rendererName) {
     // This will decorate `config` and return a renderer which we might override below.
     var cellRenderer = defaultGetCell.call(this, config, rendererName);
 
-    if (config.isUserDataArea && this.getRow(config.y).hasChildren) {
+    if (config.isUserDataArea && !this.grid.behavior.dataModel.dataSource.isLeafNode(config.dataCell.y)) {
```

Comment:
light refactor. Was using the relative grid coordinates as opposed to the data Coordinates